src/component/App/App.module.css

```diff
@@ -0,0 +1,7 @@
+.App {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+}
```

Comment:
#### [OPTIONAL]

- [ ] I highly recommend that you move all styling for your components to an isolated stylesheet folder, the component folder should only consist of components and you can simply import your styles to their respective components.